test/acceptance/features/bindKnativeService.feature

```diff
@@ -0,0 +1,46 @@
+@dev
+Feature: Bind knative service to a service
+
+  As a user of Service Binding Operator
+  I want to bind knative service to services it depends on
+
+  Background:
+    Given Namespace [TEST_NAMESPACE] is used
+    * Service Binding Operator is running
+    * PostgreSQL DB operator is installed
+
+  Scenario: Bind an imported quarkus app which is deployed as knative service to PostgreSQL database
+    Given Openshift Serverless Operator is running
+    * Knative serving is running
+    * DB "db-demo-knative" is running
+    * Quarkus application "knative-app" is imported as Knative service
+    When Service Binding Request is applied to connect the database and the application
+      """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+              name: binding-request-knative
+            spec:
+              applicationSelector:
+                group: serving.knative.dev
+                version: v1beta1
+                resource: services
+                resourceRef: knative-app
+              backingServiceSelector:
```

Comment:
backingServiceSelector is going to be deprecated. 

We should use backingServiceSelectors

Reply:
I think the API will change collectively from backingserviceSelector/selectors to services. I don't think we would be deprecating backingServiceSelector separately.

Reply:
I tried with backingServiceSelectors and sbr fails